### DISCUSSION TITLE
Problema Kata0 install ipywidgets

### DISCUSSION THREAD

**Discussion by omarfrias-maker** [LaunchX-InnovaccionVirtual/CursoIntroPython#173] [Q&A]
Hola.
  Estoy preparando mi entorno de trabajo y me sale el siguiente error.  Trabajo en una Macbook Air y ya actualice aPython.

Espero alguien me pueda ayudar.
Saludos!!

![Captura de Pantalla 2022-02-11 a la(s) 21 27 59](https://user-images.githubusercontent.com/62725469/153695141-0720adf8-375d-49b6-af6b-8c633b196ccd.png)

**Comment by Laura-Karina-GR** [comment]
Hola, también tengo problemas en esa kata, perdón por no ser de mucha ayuda, soy nueva en esto :(

**Comment by JosueVerAlar** [comment]
Es posible que necesites reiniciar el kernel. También puedes probar directamente en Visual Studio Code creando un notebook, solo revisa que tengas elegido el kernel de python. Saludos!

**Reply by omarfrias-maker** [reply]
Lo voy a intentar , gracias.

**Comment by YoelVann** [comment]
si estás en windows usa el siguiente comando:

`pip install --user ipywidgets`

a mí me funcionó.

**Reply by omarfrias-maker** [reply]
Uso MAC , lo ejecuté desde Terminal y nada me aparece el mismo error.
Se agradece de todas maneras.
